syz-fuzzer/fuzzer.go

```diff
@@ -285,19 +278,55 @@ func main() {
 		fuzzer.execOpts.Flags |= ipc.FlagEnableCoverageFilter
 	}
 
+	var files map[string][]string
+	globs := getGlobs(target)
+	if len(globs) != 0 {
+		files, err = host.CollectDirsInfo(globs)
+		if err != nil {
+			log.Fatalf("faield to collect files info: %v", err)
+		}
+	}
+
 	log.Logf(0, "starting %v fuzzer processes", *flagProcs)
 	for pid := 0; pid < *flagProcs; pid++ {
 		proc, err := newProc(fuzzer, pid)
 		if err != nil {
 			log.Fatalf("failed to create proc: %v", err)
 		}
+		proc.fuzzer.target.Files = files
```

Comment:
Don't all procs share the target? If so we need to install Files only once.
Also we can't do it when we start "go proc.loop()" for some procs, because it leads to a data race.